android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,13 +1,15 @@
-• GPX import support
-• New OpenStreetMap data as of June 2
-• MRGS and UTM coordinates support (tap on coordinates to change)
-• Improved car and bicycle routing
-• Fixed bug that prevented upload of OSM edits when on battery power
-• Now compass works on devices without gyroscope
-• Fixed missing Wiki descriptions
-• Decreased TTS (Text-To-Speech) voice speed
-• Added Fish Pass
-• Fixed funicular and parcel locker icons
-• Corrected be, eu, en, en-GB, and uk translations
+• Fresh OpenStreetMap data as of July 7, 2023
+• Fix several crashes
+• Support altitude, OSMand and Garmin colors in imported GPX tracks
+• Fixed pedestrian ETA for big slopes
+• Better car routing
+• Improved search
```

Comment:
ditto, to generic

Reply:
ditto, blame Google for 500 character limits for release notes. iOS users will see full details:
```
Search
• Improved search results ranking
• Fixed shop categories search
• Fixed airport search results with empty names

```